<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SP1Open</class>
 <widget class="QDialog" name="SP1Open">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>1-Dimensional Spectrum</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTreeWidget" name="wintree">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Available windows</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="mover">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>270</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="movel">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>250</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="peaktree">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>181</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>peaks</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="xsinc">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>200</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&lt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="xsdec">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>180</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ysdec">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>130</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ysinc">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>110</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QLabel" name="leglabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="lrb">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to window (Ins)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rrb">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove from window (Del)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Just move marker (Home)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="peakname">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>160</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Peak Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PageUp - place left peak marker</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PageDown - place right marker</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>699</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Read source</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
